src/core/ghosts.cpp

```diff
@@ -161,7 +161,7 @@ void prepare_send_buffer(GhostCommunication *gc, int data_parts) {
       for (int p = 0; p < np; p++) {
         Particle *pt = &part[p];
         if (data_parts & GHOSTTRANS_PROPRTS) {
-          memmove(insert, &pt->p, sizeof(ParticleProperties));
+          memcpy(insert, &pt->p, sizeof(ParticleProperties));
```

Comment:
I think it would be cleaner to use placement new here. The `memcpy` will just silently break if on of the types happens to be not trivially copyable, and it should be as efficient when it is.